src/category/categoryItems.ts

```diff
@@ -4,10 +4,22 @@ import { CategoryItem } from './item'
 
 const getCode = (item: CategoryItem): string => item.props.code ?? ''
 
-const getLabelOrCode = (item: CategoryItem, lang: LanguageCode) =>
-  Strings.defaultIfEmpty(getCode(item))(item.props.labels?.[lang])
+const getLabel = (item: CategoryItem, lang: LanguageCode): string => item.props.labels?.[lang] ?? ''
```

Comment:
These three functions seem similar, maybe is worth combining them and getting the output based on a parameter

Reply:
how can the generic function be named?